### PR TITLE
Fix BUG: epochs.plot_image(evoked=False) doesn't show time in seconds #88…

### LINKED ISSUE #8887: epochs.plot_image(evoked=False) doesn't show time in seconds
`epochs.plot_image(evoked=False)` doesn't show time in seconds on the x-axis but samples. If `evoked=True` (default), the x-axis of the evoked response is used, which is in seconds.

### PR DESCRIPTION
**Pull request by matteoanelli**
#### Reference issue
Fixes #8887.

#### What does this implement/fix?
If evoked=False the plot now shows the x values in seconds. Additionally adapted the overlay_times accordingly.

#### Additional information
If evoked=False the plot wasn't plotting any x_label, therefore, it has been added.

Solved with @eort.

### CODE REVIEW

**Review by drammock — APPROVED** [review]

**Comment by rob-luke** [comment]
@drammock does this need to be listed in the latest.doc?

**Comment by drammock** [comment]
> @drammock does this need to be listed in the latest.doc?

yes, I would consider this a bugfix.  @matteoanelli can you add an entry to the top of the `doc/changes/latest.inc` in the "BUG" section?  You will also need to add yourself to `doc/changes/names.inc` and follow the examples near the top of the file that do special highlighting for new contributors.

**Comment by sappelhoff** [comment]
Mh, apparently https://github.com/mne-tools/mne-python/pull/9115/commits/0efd674772cb917925c597fd15daea34b7fae5c3 did not fix the issue. Need to take a closer look and perhaps ask @larsoner for help.

**Review by sappelhoff — COMMENTED** [review]
I don't really know why contributors are now referenced using the `|foo|_` style instead of the commonly used 
```
`foo`_
```

This seems to be a new thing in `latest.inc`, starting with line 130

```
|Andres Rodriguez|_ and `Adam Li`_
```

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -115,6 +115,8 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug with :func:`mne.Epochs.plot_image` where the ``x_label`` was different depending on the evoked parameter (:gh:`9115` **by new contributor** |Matteo Anelli|_)
```
Comment (larsoner) [inline doc/changes/latest.inc:118]: It's a trick to get a bold name link in RST. You need to add a `replace::` line at the top to match (see other examples)

Comment (sappelhoff) [reply doc/changes/latest.inc:118]: okay thanks, that makes sense

Comment (matteoanelli) [reply doc/changes/latest.inc:118]: Thank you, I'll do it.

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thanks @matteoanelli !

**Pull request merged**